### PR TITLE
022 Script for requirement [Policies] [External UCS] SDL informs HMI about <externalConsentStatus> via GetListOfPermissions response

### PR DESCRIPTION
**Pull request by maria-mih**
022_ATF_N_OnAppPermissionConsent_not_sent_when_no_PTU.lua 

Test set available in #1165
Applicable json available at files/jsons/Policies/Related_HMI_API/OnAppPermissionConsent_ptu.json

@istoimenova Please review

### CODE REVIEW

File: test_scripts/Polices/External_UCS_Informing_HMI/022_ATF_N_OnAppPermissionConsent_not_sent_when_no_PTU.lua [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [Policies] [External UCS] SDL informs HMI about <externalConsentStatus> via GetListOfPermissions response
+-- [HMI API] GetListOfPermissions request/response
+-- [HMI API] ExternalConsentStatus struct & EntityStatus enum
+-- When onAppPermission Changed should be sent
```
Comment (istoimenova) [inline test_scripts/Polices/External_UCS_Informing_HMI/022_ATF_N_OnAppPermissionConsent_not_sent_when_no_PTU.lua]: @maria-mih, please put comment like: 
"Clarification: When onAppPermission Changed should be sent"

File: test_scripts/Polices/External_UCS_Informing_HMI/022_ATF_N_OnAppPermissionConsent_not_sent_when_no_PTU.lua [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+-- For Genivi applicable ONLY for 'EXTERNAL_PROPRIETARY' Polcies
+-- Check that no OnAppPermissionConsent notification (precondition for GetListofPermissions) is not sent if PTU did not take place
+--
+-- 1. Used preconditions
+-- SDL is built with External_Proprietary flag
+-- SDL and HMI are running
+-- Device is not consented and PTU does not take place
+--
+-- 2. Performed steps
+-- SDL sends to HMI BasicCommunication.ActivateApp
```
Comment (istoimenova) [inline test_scripts/Polices/External_UCS_Informing_HMI/022_ATF_N_OnAppPermissionConsent_not_sent_when_no_PTU.lua]: @maria-mih, the step should be as example:
Activate app to trigger PTU.

File: test_scripts/Polices/External_UCS_Informing_HMI/022_ATF_N_OnAppPermissionConsent_not_sent_when_no_PTU.lua [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+-- SDL sends to mobile OnHMIStatus 
+-- HMI sends to SDL BasicCommunication.ActivateApp
+--
+-- Expected result:
```
Comment (istoimenova) [inline test_scripts/Polices/External_UCS_Informing_HMI/022_ATF_N_OnAppPermissionConsent_not_sent_when_no_PTU.lua]: @maria-mih, please add comment that PTU is trigger in expected result.

File: test_scripts/Polices/External_UCS_Informing_HMI/022_ATF_N_OnAppPermissionConsent_not_sent_when_no_PTU.lua (hunk 1 above)
Comment (maria-mih) [inline test_scripts/Polices/External_UCS_Informing_HMI/022_ATF_N_OnAppPermissionConsent_not_sent_when_no_PTU.lua]: @istoimenova Done

File: test_scripts/Polices/External_UCS_Informing_HMI/022_ATF_N_OnAppPermissionConsent_not_sent_when_no_PTU.lua (hunk 2 above)
Comment (maria-mih) [inline test_scripts/Polices/External_UCS_Informing_HMI/022_ATF_N_OnAppPermissionConsent_not_sent_when_no_PTU.lua]: @istoimenova Done

File: test_scripts/Polices/External_UCS_Informing_HMI/022_ATF_N_OnAppPermissionConsent_not_sent_when_no_PTU.lua (hunk 3 above)
Comment (maria-mih) [inline test_scripts/Polices/External_UCS_Informing_HMI/022_ATF_N_OnAppPermissionConsent_not_sent_when_no_PTU.lua]: @istoimenova Added

**Pull request merged**